test/support/page_objects/common.js

```diff
@@ -304,10 +304,10 @@ export default class Common {
   }
 
   findTestSubject(selector) {
-    this.debug('in findTestSubject: ' + testSubjSelector(selector));
+    this.debug(`in findTestSubject: ${selector}`);
```

Comment:
It's possible, but there are going to be scenarios where it would be impractical to specify a totally unique data-test-subj, like inside of a commonly reused component. Perhaps we establish a guideline to only used nested selectors when necessary, and keep the ability to use nested selectors when needed.